### PR TITLE
Updated inputs, transforms and readme

### PR DESCRIPTION
**Pull request by J-C-B**
Corrected suricata in transforms for "signature field"
Updated readme to make more step by step
Created example inputs.conf with default loactions and sourcetypes

### CODE REVIEW

File: default/transforms.conf [hunk 1]
```diff
@@ -127,7 +127,7 @@ REGEX  = Priority\:\s+([^\]]+)
 FORMAT = severity_id::$1
 
 [opnsense_suricata_signature]
-REGEX  = suricata\[[^\]]+\]\:\s+\[[^\]]+\]\s+([^\]]+)\s+\[
+REGEX  = suricata\[[^\]]+\]\:\s+\[[^\]]+\]\s+\[[^\]]+\]([^\]]+)\[
```
Comment (ZachChristensen28) [inline default/transforms.conf]: Which version of suricata are you using? The proposed regex does not capture anything on my instances.

File: README.md [hunk 2]
```diff
@@ -23,17 +23,11 @@ Version 1.2.1
...
-opnsense:suricata
-opnsense:squid
-opnsense:cron
-opnsense:unbound
-opnsense:lighttpd
-opnsense:access
-```
+## Universal Forwarder
+
+Use the FreeBSD universal forwarder on OpnSense at [splunk.com](https://www.splunk.com/en_us/download/universal-forwarder.html#tabs/freebsd)
```
Comment (ZachChristensen28) [inline README.md]: I would advise against putting a UF on the firewall itself. Typically, syslog is utilized to send the logs to a syslog server which Splunk is monitoring.

File: README.md [hunk 3]
```diff
@@ -45,18 +39,28 @@ Heavy Forwarders | Yes | Conditional | Not required.
...
-Set the sourcetype to "opnsense" in the inputs.conf file on the forwarder.
+Set the sourcetype to "opnsense" in the inputs.conf file on the forwarder, the actual data type will be automatically identified on ingestion.
 
 i.e.
 
 ```
 # Sample inputs.conf
 
-[monitor:///var/log/firewall.log]
+[monitor:///var/log/filter.log]
```
Comment (ZachChristensen28) [inline README.md]: If using a syslog server, this file could be named whatever and not filter or firewall. Which is why this is an example.

File: default/transforms.conf (hunk 1 above)
Comment (J-C-B) [reply default/transforms.conf]: I'm running;

OPNsense 20.1-amd64
FreeBSD 11.2-RELEASE-p16-HBSD
OpenSSL 1.1.1d 10 Sep 2019

in the package manager it shows

suricata | 4.1.6 | 6.03MiB | GPLv2

File: README.md (hunk 3 above)
Comment (J-C-B) [reply README.md]: OK cool, it wasn't clear on what was expected from the readme so went with UF, ignore this if you disagree but suggest linking to docs for syslog configuration for new users

also thanks so much for this TA - its fantastic

File: README.md (hunk 2 above)
Comment (J-C-B) [reply README.md]: OK cool, it wasn't clear on what was expected from the readme so went with UF, ignore this if you disagree but suggest linking to docs for syslog configuration for new users

Comment (ZachChristensen28) [reply README.md]: That makes sense. I can put a brief walk-through on how to send data to a syslog server. Thanks for the feedback on this.

File: README.md (hunk 3 above)
Comment (ZachChristensen28) [reply README.md]: Gotcha. I will try to make this a little more clear for new users. Thank you for your input on this.

File: default/transforms.conf (hunk 1 above)
Comment (ZachChristensen28) [reply default/transforms.conf]: Thank you. I seem to be on the same version as you. Can you provide me with a few more data points?
If you navigate to `Services > Intrusion Detection > Administration > (Settings Tab) > Right below the Setting tab, toggle the "Advanced Mode" switch`:
- Is the "Enable syslog alerts" box checked?
- Is the "Enable eve syslog output" box checked?
- Is the "Log package payload" box checked?

Comment (J-C-B) [reply default/transforms.conf]: Hi - info below;
    IPS mode = disabled (non blocking)
    Promiscuous mode = disabled
    Is the "Enable syslog alerts" box checked? - Yes
    Is the "Enable eve syslog output" box checked? - Yes
    Is the "Log package payload" box checked? - No

Example event below 
`Mar  3 16:24:02 OPNsense suricata[25620]: [wDrop] [1:2012647:5] ET POLICY Dropbox.com Offsite File Backup in Use [Classification: Potential Corporate Privacy Violation] [Priority: 1] {TCP} 162.125.136.1:443 -> 192.168.0.176:65289 `

**Comment by ZachChristensen28** [comment]
I am closing this pull request. I will be incorporating your feedback into the next release which will be committed shortly.

**Pull request closed without merging**